client/blocks/user-mention-suggestions/docs/example-input.jsx

```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ExampleInput from './example-input';
+import withUserMentionSuggestions from '../with-user-mention-suggestions';
+
+const UserMentionSuggestionsExampleInput = ( { onKeyPress } ) => (
```

Comment:
Did we need to add an explicit displayName here for this example component too?

Reply:
It'll only ever be used inside `UserMentionSuggestionsExample` (to demonstrate how the HoC works), so should be alright without one I think.